crates/bevy_reflect/src/impls/std.rs

```diff
@@ -1484,6 +1485,11 @@ impl_reflect! {
     }
 }
 
+impl_reflect! {
```

Comment:
Any reason we're not using `impl_reflect_value!` here? I feel like this makes more sense as a `Value` type, no?

Reply:
I got an error when doing that. I can take another stab at it.

Reply:
What was the error?

Reply:
It was something about PhantomData not being able to provide compilation information. I don't get it now though.